lib/routes/password.js

```diff
@@ -313,6 +325,23 @@ module.exports = function (
         if (butil.buffersAreEqual(passwordForgotToken.passCode, code) &&
             passwordForgotToken.ttl() > 0) {
           db.forgotPasswordVerified(passwordForgotToken)
+            .then(
+              function (accountResetToken) {
+                return db.account(passwordForgotToken.uid)
```

Comment:
Having to hit the db again here to lookup the email is not ideal.

The `passwordForgotToken` class definition claims to have an `email` property which should be populated when we do the initial credentials check.  Was it missing in your testing, or was it just not obvious that it existed?  Either way, we should make it so you can just do `passwordForgotToken.email` rather than calling `db.account()`.